docs/contracts/ROUTING_POLICY.md

```
@@ -0,0 +1,54 @@
+
+## Patterns
+
+| Allowed | Forbidden |
+|---|---|
+| `client → gate` | `node_a → node_b` |
+| `node → gate` | `orchestrator → worker (direct)` |
+| `gate → worker` | `worker → worker` |
+
+## Selection Algorithm (Gate)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)